stopwatch/stopwatch.go

```diff
@@ -46,6 +46,7 @@ type ResetMsg struct {
 // Model for the stopwatch component.
 type Model struct {
 	d       time.Duration
+	sT      time.Time
```

Comment:
i would name this `start` instead 🤔 

```suggestion
	start   time.Time
```